packages/store/src/atoms/tracks.ts

```diff
@@ -20,4 +20,5 @@ export interface Track {
     title: string;
     problems: string[];
     description: string;
+    hidden: boolean;
```

Comment:
Frontend should never know about this field

Reply:
Made the change